### PR TITLE
F-Secure.WithSecureElements version 1.2.0.0

### PR DESCRIPTION
**Pull request by MateuszKaczm**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80866)

### CODE REVIEW

**Comment by MateuszKaczm** [comment]
@microsoft-github-policy-service agree [company="{private}"]

**Comment by MateuszKaczm** [comment]
@microsoft-github-policy-service agree

**Comment by MateuszKaczm** [comment]
@microsoft-github-policy-service agree [company="{}"]

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.installer.yaml:5]:
```suggestion
PackageVersion: 4.36.3020.0
```
This is the PackageVersion of the MSI

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+PackageLocale: en-US
+Publisher: F-Secure
+PackageName: WithSecure_Elements
+License: Annual License
+ShortDescription: Antivirus/Security
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml:10]: Please provide a better short description

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.36.3020.0
```
This is the PackageVersion of the MSI

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.yaml:5]:
```suggestion
PackageVersion: 4.36.3020.0
```
This is the PackageVersion of the MSI

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://download.sp.f-secure.com/PSB/latest/installer/OfflineInstallerCP-PSB1.msi
+  InstallerSha256: 3A1441E961BDB6C53BFAA83B74CC982738D10F9D1132DFAE14CC72F5A1F34465
+  ProductCode: '{D2668EA0-12FC-4DCD-BFCF-00B5B43B3CBA}'
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.installer.yaml:12]:
```suggestion
  AppsAndFeaturesEntries: 
  - DisplayName: WithSecure™ Elements Agent
    ProductCode: '{D2668EA0-12FC-4DCD-BFCF-00B5B43B3CBA}'
    DisplayVersion: 22.5
```

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+PackageLocale: en-US
+Publisher: F-Secure
+PackageName: WithSecure_Elements
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml:8]:
```suggestion
PackageName: WithSecure Elements Agent
```

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+PackageLocale: en-US
+Publisher: F-Secure
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml:7]:
```suggestion
Publisher: F-Secure Corporation
```
This should match control panel

File: manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: F-Secure.WithSecureElements
+PackageVersion: 1.2.0.0
+PackageLocale: en-US
+Publisher: F-Secure
+PackageName: WithSecure_Elements
+License: Annual License
```
Comment (Trenly) [inline manifests/f/F-Secure/WithSecureElements/1.2.0.0/F-Secure.WithSecureElements.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/80866/files#r983038088

**Comment by ItzLevvie** [comment]
Close with reason: Replaced by PR 82820 to avoid pull request being stale;

**Pull request closed without merging**